submissions/MarharytaBoichenko/html_css_popup/sass/main.scss

```diff
@@ -0,0 +1,304 @@
+  align-content: end;
+  gap: 20px;
+  @media screen and (min-width: 768px) {
+    gap: 25px;
+  }
+}
+
+.nav__link {
+  font-size: 16px;
+  font-weight: 700;
+  color: rgb(93, 92, 92);
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+
+  @media screen and (min-width: 768px) {
+    font-size: 20px;
+  }
+}
+
+.user {
+  position: relative;
+  display: grid;
+  grid-auto-flow: column;
+  align-items: center;
+  gap: 15px;
+
+  @media screen and (min-width: 768px) {
+    gap: 25px;
+  }
+}
+
+.user__item:not(:last-child) {
+  margin-right: 15px;
+}
+
+.user__menu {
+  display: block;
+  width: 35px;
+  height: 35px;
+  background-image: url('../popup-icons/popup-button.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.menu {
+  display: none;
+  position: relative;
+}
+
+.user__checkbox {
+  position: absolute;
+  width: 35px;
+  height: 35px;
+  cursor: pointer;
+  appearance: none;
+  outline: none;
+}
+
+.user__checkbox:checked ~ .menu {
+  position: absolute;
+  right: 0;
+  top: 55px;
+  display: block;
+  width: 350px;
+  height: 450px;
+  padding: 25px;
+  overflow-y: auto;
+  background-color: $bcgColor;
+  border-radius: 3px;
+  box-shadow: 16px 16px 16px 16px rgba(211, 208, 212, 0.75);
+
+  @media screen and (min-width: 768px) {
+    width: 450px;
+  }
+}
+
+// .menu::after {
+//   content: '';
+//   display: none;
+//   width: 15px;
+//   height: 15px;
+//   background-color: $bcgColor;
+//   transform: rotate(45deg);
+//   position: absolute;
+//   top: 63px;
+//   right: 40%;
+// }
+
+.user__checkbox:focus ~ .user__menu {
+  border: 2px solid grey;
+  border-radius: 3px;
+}
+
+.menu__list {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+}
+
+.menu__item {
+  display: grid;
+  justify-items: center;
+}
+
+.menu__item--more {
+  position: relative;
+  display: grid;
+  grid-column: 1 / 4;
+  width: 100%;
+}
+
+.menu__link {
+  width: 100%;
+  font-size: 12px;
+  text-align: center;
+  padding: 15px 0;
+  display: grid;
+  justify-items: center;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  &:hover,
+  &:focus {
+    border: 1px solid grey;
+    background-color: rgb(180, 228, 254);
+    outline: none;
+    cursor: pointer;
+  }
+
+  @media screen and (min-width: 768px) {
+    font-size: 14px;
+  }
+}
+
+.menu__icon {
+  max-height: 45px;
+  padding-bottom: 15px;
+
+  @media screen and (min-width: 768px) {
+    max-height: 50px;
+  }
+}
+
+.user__notification {
+  width: 25px;
+  height: auto;
+}
+
+.user__photo {
+  width: 40px;
+  height: auto;
+}
+
+.more {
+  position: relative;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+}
+
+.more-list {
+  display: none;
+}
+
+.more-checkbox {
+  position: absolute;
+  top: 35px;
+  width: 100%;
+  height: 20px;
+  cursor: pointer;
+  appearance: none;
+  outline: none;
+}
+
+.more-checkbox:focus ~ .label {
+  color: white;
```

Comment:
Prefer not to use it, better hex model, `#FFF` as you did it earlier

Reply:
ok